test/e2e/common/node/security_context.go

```diff
@@ -1001,8 +1001,8 @@ var _ = SIGDescribe("User Namespaces for Pod Security Standards [LinuxOnly]", fu
 	f := framework.NewDefaultFramework("user-namespaces-pss-test")
 	f.NamespacePodSecurityLevel = admissionapi.LevelRestricted
 
-	ginkgo.Context("with UserNamespacesSupport and UserNamespacesPodSecurityStandards enabled", func() {
-		f.It("should allow pod", feature.UserNamespacesPodSecurityStandards, framework.WithFeatureGate(features.UserNamespacesSupport), framework.WithFeatureGate(features.UserNamespacesPodSecurityStandards), func(ctx context.Context) {
+	ginkgo.Context("with UserNamespacesSupport enabled", func() {
+		f.It("should allow pod", framework.WithFeatureGate(features.UserNamespacesSupport), func(ctx context.Context) {
```

Comment:
It seems only this test is failing, all the other userns tests are fine. I wonder if this is what is missing?

```suggestion
		f.It("should allow pod", features.UserNamespacesSupport, framework.WithFeatureGate(features.UserNamespacesSupport), func(ctx context.Context) {
```

Reply:
ah good idea, they were separate because this one is PSA restricted, but the others can be adapted to do the same